jsonexample/build.yaml

```diff
@@ -14,13 +14,3 @@ targets:
       json_serializable|json_serializable:
         generate_for:
         - lib/json_serializable/*.dart
-        options:
```

Comment:
Did the header option get removed from source_gen/builder_runner?

Reply:
yup